### ISSUE TITLE
edfR doesn't compile on Linux

### ISSUE THREAD

**Issue by tmalsburg** [jashubbard/edfR_test#1]
I tried installing via `install_github('jashubbard/edfR')` on Ubuntu 15.04.  For some reason this seems to be using the Makevars for OSX instead of those for Linux:

```
g++ -shared -L/usr/local/lib -o edfR.so RcppExports.o edf_api.o process_eyedata.o -framework edfapi -F/Library/Frameworks/
g++: error: edfapi: No such file or directory
g++: error: unrecognized command line option ‘-framework’
```

The `configure` file also adds `/usr/local/lib` to `PKG_LIBS` although edfapi actually gets installed into `/usr/lib` (when following the instructions given in the SR Research forum).

(I'm planning to use the package over the next weeks.  So hopefully I will be able to make some contributions.)

**Comment by jashubbard** [comment]
Ok I updated configure so that it creates a new Makevars no matter what, not just when it doesn't exist. I also changed the path to /usr/lib . Give it another shot.

**Comment by tmalsburg** [comment]
Hm, same behavior as before.  Unfortunately, I don't know much about R's build process.  I'll look into it ...

**Comment by tmalsburg** [comment]
Fixed in 0052e1331bbc4d0abc11db953534bcb8b95e7fdd.  There was an error in the conditional for detecting the OS.
